bin/methods/start.js

```diff
@@ -1,23 +1,33 @@
 module.exports = start
 const { writeFile } = require('fs')
-const path = require('path')
+const { resolve } = require('path')
 const log = require('log-cb')
 const { Client } = require('discord.js')
 
-function start (flags, plugins = []) {
+function start (flags) {
   // Load configuration file
   let config = {}
-  const configPath = path.resolve(process.cwd(), 'package.json')
+  const configPath = resolve(process.cwd(), 'package.json')
```

Comment:
I didn't do this because of conflicts with the `resolve` pacakge:
```js
const { resolve } = require('path')
// vs
const resolve = require('resolve')
```
More clear with `path` there IMO.